engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -161,21 +161,47 @@ class RuntimeServerTest
             )
           )
 
-        def mainY(contextId: UUID, fromCache: Boolean = false): Api.Response =
```

Comment:
@4e6, the `RuntimeServerTest` has been mostly updated to show the change in delivered messages. Other tests are using filter like `receiveNIgnorePendingExpressionUpdates` or `receiveNIgnoreExpressionUpdates`, the `RuntimeServerTest` properly checks the delivered messages in most cases.